dpnp/dpnp_iface_mathematical.py

```diff
@@ -93,6 +93,15 @@
 ]
 
 
+def convert_result_scalar(result, keepdims):
```

Comment:
I would add `to` or `2` to the function name, `convert_result_to_scalar` or `convert_result2scalar`.

Reply:
I would vote for `convert_result2scalar` :-)